lib/ProductOpener/Redis.pm

```diff
@@ -131,6 +131,7 @@ sub push_to_redis_stream ($user_id, $product_ref, $action, $comment, $diffs) {
 				# We let Redis generate the id
 				'*',
 				# fields
+				'ts', $timestamp,
```

Comment:
Maybe just "t" ? we have been using _t as a suffix for timestamps, or maybe "timestamp" is more explicit